exopy/app/log/widgets.enaml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2018 by Exopy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Widget used to display the rotating log.
+
+"""
+from enaml.widgets.api import (Container, Dialog, Label, ScrollArea)
```

Comment:
The label gives us a grey background that does not make things extremely readable. You can either use styling to get a nicer background (not my favorite), or use a MultilineField in read_only mode (easier)